### PR TITLE
Deploying steps 3.5.0a, required by Dan

### PR DESCRIPTION
**Pull request by ferdonline**
Dan requires this version which implements intersection.
Previously I deployed it in a separate tree in proj40 but in the meantime the main functionality was merged to master featuring a lot of improvements.
Since there hasn't been a recent Steps in some time, and I'm not an authority there to create a tag, I believe an interim 3.5.0a will do (I know I know, debatable version name but... well, better options?)

### CODE REVIEW

**Comment by pramodk** [comment]
Let’s wait for the fix from @alkino on Monday and then we can deploy new version?

**Review by matz-e — APPROVED** [review]
Fine with alpha. I suppose this going to be deprecated soon, though? Otherwise that `commit=` should change to a tag or so…

**Comment by ferdonline** [comment]
> Let’s wait for the fix from @alkino on Monday and then we can deploy new version?

Sounds good to me. Weren't they merged already?

**Comment by ferdonline** [comment]
> Fine with alpha. I suppose this going to be deprecated soon, though? Otherwise that `commit=` should change to a tag or so…

Yep, this should be interim.

File: var/spack/repos/builtin/packages/steps/package.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Steps(CMakePackage):
     git      = "git@github.com:CNS-OIST/HBP_STEPS.git"
 
     version("develop", branch="master", submodules=True)
-    version("3.5.0a",  commit="1c8a5b0", submodules=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/steps/package.py]: Is this actually getting re-built? I recall commit hash changes not changing the hash of the package… :/

**Comment by pramodk** [comment]
@alkiko is about to merge PR. We should include that.

**Comment by ferdonline** [comment]
> @alkiko is about to merge PR. We should include that.

It is merged

File: var/spack/repos/builtin/packages/steps/package.py (hunk 1 above)
Comment (ferdonline) [reply var/spack/repos/builtin/packages/steps/package.py]: Humm... indeed it seems that even though tests re-run, it didn't recompile Steps.
But is it problematic? only difference is it's rebuilding another commit, which has been tested

Comment (matz-e) [reply var/spack/repos/builtin/packages/steps/package.py]: Something to keep in mind. Unexpected effects, and why I'm not a fan of moving versions/commits around.

**Pull request merged**